imp_developer.py

```diff
@@ -1602,13 +1604,25 @@ def on_new_device_group_name_provided(self, name):
         self.update_settings(EI_DEPLOYMENT_ID, EI_DEPLOYMENT_NEW)
         self.on_action_complete()
 
+class ImpSelectDeviceCommand(BaseElectricImpCommand):
```

Comment:
Not to introduce new concepts, I would move the method to BaseElectricImpCommand